components/certificate-mgt/org.wso2.carbon.identity.certificate.management.core/src/main/java/org/wso2/carbon/identity/certificate/management/core/constant/CertificateMgtConstants.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.certificate.management.core.constant;
+
+/**
+ * Constants for Certificate Management.
+ */
+public class CertificateMgtConstants {
+
+    public static final String NAME_FIELD = "Name";
+    public static final String CERTIFICATE_FIELD = "Certificate";
+
+    /**
+     * Enum for error messages.
+     */
+    public enum ErrorMessages {
+
+        // Client errors.
+        ERROR_CERTIFICATE_DOES_NOT_EXIST("60001", "Unable to perform the operation.",
```

Comment:
These error codes shouldn't conflict with existing error codes right ?

Reply:
Better to prefix the error code. Ex: CMS-xxxxx

Reply:
Currently the prefix is added from the REST api layer. As this is used as a internal component and the exceptions from this components should be properly handled and wrapped in the other components, error code conflicts should not be occurred. Hence we can have the error codes as it is.